test/unit/util/handlers/BooleanHandler.test.ts

```diff
@@ -13,8 +13,8 @@ describe('A BooleanHandler', (): void => {
     handlerFalse = new StaticAsyncHandler(true, false);
     handlerTrue = new StaticAsyncHandler(true, true);
     handlerError = new StaticAsyncHandler(true, null) as any;
-    handlerError.handle = (): never => {
-      throw new InternalServerError();
+    handlerError.handle = async(): Promise<never> => {
+      throw new Error('handleError');
```

Comment:
Why this change? (Because in general our handlers should throw HTTP errors anyway).

Reply:
I had some trouble with it earlier so changed it for traceability; problem turned out to be the sync throwing. Will re-adjust.